<ui version="4.0" >
 <class>ScriptConsole</class>
 <widget class="QWidget" name="ScriptConsole" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Script Console</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" >
   <item row="0" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Input</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7" >
       <item row="0" column="0" colspan="4" >
        <widget class="QStackedWidget" name="stackInput" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="P1" >
          <layout class="QGridLayout" name="gridLayout_6" >
           <item row="0" column="0" >
            <widget class="QPlainTextEdit" name="input" >
             <property name="plainText" >
              <string># Type your script right here
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="P2" >
          <layout class="QGridLayout" name="gridLayout_4" >
           <property name="margin" >
            <number>0</number>
           </property>
           <item row="0" column="0" >
            <widget class="QListWidget" name="scriptsList" >
             <property name="sortingEnabled" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="loadButton" >
         <property name="text" >
          <string>load</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="saveButton" >
         <property name="text" >
          <string>save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>468</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" >
        <widget class="QPushButton" name="execButton" >
         <property name="text" >
          <string>execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QToolBox" name="toolBox" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="page" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>368</width>
            <height>369</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Standard Output</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout" >
           <item row="0" column="0" colspan="2" >
            <widget class="QTextBrowser" name="stdOut" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>290</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" >
            <widget class="QPushButton" name="clearButton" >
             <property name="text" >
              <string>clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>368</width>
            <height>369</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Standard Error</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2" >
           <item row="0" column="0" >
            <widget class="QTextBrowser" name="stdErr" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>424</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptConsole</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>937</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel" >
     <x>491</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>stdOut</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>919</x>
     <y>610</y>
    </hint>
    <hint type="destinationlabel" >
     <x>790</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>40</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>execButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel" >
     <x>396</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
